src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/Licence.kt

```diff
@@ -254,260 +215,16 @@ class Licence(
-      supersededDate,
-      submittedDate,
-      dateCreated,
-      dateLastUpdated,
-      updatedByUsername,
-      standardConditions,
-      additionalConditions,
-      bespokeConditions,
-      responsibleCom,
-      submittedBy,
-      createdBy,
-      variationOfId,
-      versionOfId,
-      licenceVersion,
-    )
-  }
-
-  override fun toString(): String {
-    return "Licence(" +
-      "id=$id, " +
-      "kind=$kind, " +
-      "typeCode=$typeCode, " +
-      "version=$version, " +
-      "statusCode=$statusCode, " +
-      "nomsId=$nomsId, " +
-      "bookingNo=$bookingNo, " +
-      "bookingId=$bookingId, " +
-      "crn=$crn, " +
-      "pnc=$pnc, " +
-      "cro=$cro, " +
-      "prisonCode=$prisonCode, " +
-      "prisonDescription=$prisonDescription, " +
-      "prisonTelephone=$prisonTelephone, " +
-      "forename=$forename, " +
-      "middleNames=$middleNames, " +
-      "surname=$surname, " +
-      "dateOfBirth=$dateOfBirth, " +
-      "conditionalReleaseDate=$conditionalReleaseDate, " +
-      "actualReleaseDate=$actualReleaseDate, " +
-      "sentenceStartDate=$sentenceStartDate, " +
-      "sentenceEndDate=$sentenceEndDate, " +
-      "licenceStartDate=$licenceStartDate, " +
-      "licenceExpiryDate=$licenceExpiryDate, " +
-      "licenceActivatedDate=$licenceActivatedDate, " +
-      "topupSupervisionStartDate=$topupSupervisionStartDate, " +
-      "topupSupervisionExpiryDate=$topupSupervisionExpiryDate, " +
-      "probationAreaCode=$probationAreaCode, " +
-      "probationAreaDescription=$probationAreaDescription, " +
-      "probationPduCode=$probationPduCode, " +
-      "probationPduDescription=$probationPduDescription, " +
-      "probationLauCode=$probationLauCode, " +
-      "probationLauDescription=$probationLauDescription, " +
-      "probationTeamCode=$probationTeamCode, " +
-      "probationTeamDescription=$probationTeamDescription, " +
-      "appointmentPerson=$appointmentPerson, " +
-      "appointmentTime=$appointmentTime, " +
-      "appointmentAddress=$appointmentAddress, " +
-      "appointmentContact=$appointmentContact, " +
-      "spoDiscussion=$spoDiscussion, " +
-      "vloDiscussion=$vloDiscussion, " +
-      "approvedDate=$approvedDate, " +
-      "approvedByUsername=$approvedByUsername, " +
-      "approvedByName=$approvedByName, " +
-      "supersededDate=$supersededDate, " +
-      "submittedDate=$submittedDate, " +
-      "dateCreated=$dateCreated, " +
-      "dateLastUpdated=$dateLastUpdated, " +
-      "updatedByUsername=$updatedByUsername, " +
-      "standardConditions=$standardConditions, " +
-      "additionalConditions=$additionalConditions, " +
-      "bespokeConditions=$bespokeConditions, " +
-      "responsibleCom=$responsibleCom, " +
-      "submittedBy=$submittedBy, " +
-      "createdBy=$createdBy, " +
-      "variationOfId=$variationOfId, " +
-      "versionOfId=$versionOfId, " +
-      "licenceVersion=$licenceVersion" +
-      ")"
+    return Objects.hash(id)
```

Comment:
Take it this was what you were talking about re the hash and equals using the id only?